projects/plugins/jetpack/modules/theme-tools/content-options/featured-images-fallback.php

```diff
@@ -63,9 +63,10 @@ function jetpack_featured_images_fallback_get_image( $html, $post_id, $post_thum
 
 			$image_sizes = '';
 
+			$width  = isset( $image['width'] ) ? intval( $image['width'] ) : null;
+			$height = isset( $image['height'] ) ? intval( $image['height'] ) : null;
+
 			if ( $image['src_width'] && $image['src_height'] && $image['width'] && $image['height'] ) {
```

Comment:
We also have an
```
Undefined array key "src_width"
```
on this line, should we add an `isset( $image['src_width'] )` here?

Reply:
Probably more like
```suggestion
			if ( ! empty( $image['src_width'] ) && ! empty( $image['src_height'] ) && ! empty( $image['width'] ) && ! empty( $image['height'] ) ) {
```